build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -246,8 +246,7 @@ flowExporter:
 nodePortLocal:
 {{- with .Values.nodePortLocal }}
 # Enable NodePortLocal, a feature used to make Pods reachable using port forwarding on the host. To
-# enable this feature, you need to set "enable" to true, and ensure that the NodePortLocal feature
-# gate is also enabled (which is the default).
+# enable this feature, you need to set "enable" to true.
```

Comment:
Since the default will be true for a GA feature, this line should be `disable this feature, you need to set "enable" to false.`

Reply:
Done.